nodejs/templates/deployment.yaml

```diff
@@ -20,6 +20,8 @@ spec:
         name: {{ template "hmcts.releaseName" . }}
         {{- if .Values.environment }}
```

Comment:
this if statement will need moving right?

Reply:
it doesn't make sense after this change, but why before, the current one looks fine?

Reply:
true I would expect it to be a loop of 0 values and do nothing, but if its nil maybe it fails? so might still need that if check for the environment section?

Reply:
can you test what happens with no environment values? and if its fine we can merge otherwise needs a change

Reply:
I tested it 1) without having environment  2) having environment: but no values - making it environment: null in computed values . It works in both cases. Happy to add if we think it makes it more readable.

Reply:
nah is fine thanks